### DISCUSSION TITLE
Seems to have found a Bug in Neper 4.5.1-4

### DISCUSSION THREAD

**Discussion by smallrain1996** [neperfepx/neper#673] [Q&A] [answered]
Here is my test code, which works fine, but it doesn't make sense because there is a missing Element set in the INP file, FACE252
```
name=test_neper_2D
number=300
mkdir $name; cd $name
neper -T -n $number -dim 2 \
-periodicity 'x,y' \
-group "id<=150?1:2" \
-stattess dim,cellnb,x,y,z,size \
-statcell id,group,size,diameq \
-o $name
neper -M $name.tess -format inp,msh,phase \
-nset edges -faset none \
-statmesh eltnb,nodenb,size \
-statelt id,x,y \
-statnode id,x,y \
-statelset id,group,elts \
-elttype quad -rcl 0.5 \
-o $name
neper -V $name.msh -datacellcol id -print $name
cd ..
zip -r $name.zip $name
```
[test_neper_2D.zip](https://github.com/neperfepx/neper/files/11073564/test_neper_2D.zip)

**Comment by smallrain1996** [comment, marked as answer]
The basic reason I probably already know, set 252 does not contain any elements, which means there are some cases where Neper's simulation will not have the same number of grains as I specified, right?

**Reply by rquey** [reply]
With the "mapped meshing" technique (which does not follow the shape of the grains), yes, it may happen...
